### ISSUE TITLE
I receive 'JavaScript error in async script.' when calling waitForAngularRequestsToFinish()

### ISSUE THREAD

**Issue by kermek** [paul-hammant/ngWebDriver#7]
May be this is connected with Bootstrapped AngularJS:
http://stackoverflow.com/questions/25596706/using-protractor-test-with-bootstrapped-angularjs

**Comment by manoj9788** [comment]
@kermek  Do you still get this error ? I think it might be an temporary issue. 
We can close this issue based on your comments.

**Comment by kermek** [comment]
I couldn't not solve this issue. So, I decided to not use this library since it is not working for me(

**Comment by manoj9788** [comment]
Can we please see the test that you were running ?
If possible a test proj in Github. we don't get to see any errors and all our tests works fine. So just curious where it fails

**Comment by kermek** [comment]
I believe that this is not a matter of test I am running, but rather web application I am testing.
Angular bootstrapped, so I receive 'org.openqa.selenium.WebDriverException: javascript error: Cannot read property 'get' of undefined'.
I modified JS to use bootrstrapped angular, but the '$browser' variable is not existing in my case.

Since the whole concept of such usage is not documented I decided to workaround it and not use this library.
